tests/lib/workspace.py

```diff
@@ -124,7 +124,8 @@ def _ssh_agent(self):
             raise
 
     def execute(self, command: str, capture=False, check=True,
-                log_stdout=True, log_stderr=True, env=None,
+                log_stdout=True, log_stderr=True,
```

Comment:
`log_stdout: bool =True` and same for `log_stderr` given that the commit msg says that you are extending typehints for workspace.execute.

Reply:
I thought mypy was able to imply types from the given True/False?

Reply:
I don't know. But just look at the 2nd line of https://www.python.org/dev/peps/pep-0020/#id2 :)